package.json

```diff
@@ -4,7 +4,7 @@
   "description": "",
   "main": "webpack.config.js",
   "dependencies": {
-    "auth0-lock": "^10.4.0",
+    "auth0-lock": "10.4.0",
```

Comment:
Wouldn't the caret make them less specific? (10.x.x) You could add a tilde to only get current version updates if you want (10.4.x)